apps/emqx_management/src/emqx_mgmt_data_backup.erl

```diff
@@ -476,6 +477,15 @@ import_cluster_hocon(BackupDir, Opts) ->
             #{}
     end.
 
+upgrade_raw_conf(SchemaMod, RawConf) ->
+    case erlang:function_exported(SchemaMod, upgrade_raw_conf, 1) of
```

Comment:
To ensure this is really loaded and the check for exported functions work properly (happened in the past with other similar calls :sweat_smile: ):

```suggestion
    _ = SchemaMod:module_info(),
    case erlang:function_exported(SchemaMod, upgrade_raw_conf, 1) of
```

Reply:
Fixed :slightly_smiling_face: